### DISCUSSION TITLE
Upgrade to Mantine v7?

### DISCUSSION THREAD

**Discussion by Rigel772** [KevinVandy/mantine-react-table#144] [General]
Any plans for that in near future?

**Comment by ashkyn** [comment]
> Any plans for that in near future?

#68 Discussion about Mantine v7 started here; work has begun on the refactor.

**Comment by KevinVandy** [comment]
You can join the discord if interested in helping with the Mantine v7 upgrade. It's in progress

**Reply by eluce2** [reply]
The mantine discord or another one? My team had built [our own implementation](https://github.com/proofgeist/mantine-tanstack-table) of this idea before we discovered this repo. I'm also willing to help if I can

**Reply by eluce2** [reply]
Ah, I found it. Will post there instead :)

**Comment by KevinVandy** [comment]
https://github.com/KevinVandy/mantine-react-table/discussions/169
